Web/Admin.UI/src/app/components/core/vd-button/vd-button.component.html

```
@@ -0,0 +1,20 @@
+<ng-container *ngIf="type === 'link'; else buttonTemplate">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)